tests/trainer/test_checkpoint.py

```diff
@@ -295,6 +298,86 @@ def test_save_overwrite(
     trainer.fit()
 
 
+pytest.mark.timeout(90)
+
+
+def test_checkpoint_with_object_store_logger(
```

Comment:
Can you parameterize the `load_object_store` to be a object store or a logger destination, so we can have test coverage for this change?

Reply:
I agree with this suggestion, but I want to punt it to next PR because I'll be refactoring the checkpoint tests anyways (ideally the parameterization just goes into the big existing test, and most of this gets deleted). Originally wrote this before the refactor decision, and imo it's cleaner to do it all in a separate PR so code review is easier